src/index.d.ts

```diff
@@ -49,5 +53,6 @@ declare module 'roo-ui' {
       greys: KeyValue<string>;
       ui: KeyValue<string>;
     };
+    textStyles: NestedKeyValue<string>;
```

Comment:
Actually I think this might be a more accurate type

```suggestion
    textStyles: KeyValue<NestedKeyValue<string>>;
```

Reply:
`NestedKeyValue<string>` allows for

```
const textStyles = { caps: 'a string value' };
``` 

But we want to enforce objects at the first level 

```
const textStyles = { caps: { textTransform:  'a string value' } };
```